src/index.js

```diff
@@ -153,3 +153,24 @@ exports.WelcomeChannel = require('./structures/WelcomeChannel');
 exports.WelcomeScreen = require('./structures/WelcomeScreen');
 
 exports.WebSocket = require('./WebSocket');
+
+// External
+exports.ActivityType = require('discord-api-types').ActivityType;
```

Comment:
feel like it would be worth it to use a variable instead of repeated `require`s 😄

Reply:
Probably not because it will most likely fuck with bundlers again, it's why we reverted all the dynamic imports in general.

Reply:
Would it really fuck bundlers up if we did `const v9 = require('discord-api-types/v9');` and export from there?

Reply:
Most likely increase bundle size for no apparent reason because of tree-shaking